AdyenNetworking/APIClient/HTTPResponse.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 public struct HTTPResponse<R: Response> {
     
     /// HTTP Headers.
-    public let headers: [String: String]
+    public let headers: [String: Any]
```

Comment:
what caused this change? can the http headers be something else other than string?

Reply:
yes, headers can have nests dictionaries :-)

Reply:
can you please give an example? i don't think we should apply any logic when parsing http headers, by design they are just two string values.

Reply:
![Screen Shot 2022-08-12 at 10 46 46](https://user-images.githubusercontent.com/2619933/184319194-a415de00-da62-40c4-a9e5-f84f4daa076a.png)

these are real headers from a real API `https://gorest.co.in/public/v1/`
```
{
  "x-content-type-options" : "nosniff",
  "x-runtime" : "0.014042",
  "x-ratelimit-reset" : "1",
  "Alt-Svc" : "h3=\":443\"; ma=86400, h3-29=\":443\"; ma=86400",
  "referrer-policy" : "strict-origin-when-cross-origin",
  "Cache-Control" : "no-cache",
  "x-download-options" : "noopen",
  "report-to" : "{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=M%2BzvAgsFCLW7s0uDS6c3%2FLA%2B%2BN8gDEAUnfeoWIFFMjeXmnkgyKgiUV3prd4CQ5yv0DIXTq01eWhsun1RJjhsYUSh%2F2qlckwVhUuhu1Vv7kN56ZjPIjuXWipHv%2BJpAss%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}",
  "Vary" : "Accept-Encoding, Origin",
  "Date" : "Fri, 12 Aug 2022 08:19:03 GMT",
  "Content-Type" : "application/json; charset=utf-8",
  "Content-Encoding" : "br",
  "x-xss-protection" : "0",
  "x-ratelimit-limit" : "90",
  "Strict-Transport-Security" : "max-age=63072000; includeSubDomains",
  "x-request-id" : "70c25f89-fa12-416b-9b50-9f76e857f5c8",
  "x-frame-options" : "SAMEORIGIN",
  "expect-ct" : "max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"",
  "x-ratelimit-remaining" : "89",
  "Server" : "cloudflare",
  "nel" : "{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}",
  "x-permitted-cross-domain-policies" : "none",
  "cf-ray" : "7397c8aa8f7b7a4c-DUS",
  "cf-cache-status" : "DYNAMIC"
}
```

Reply:
all of those values look like strings to me. can you point out exactly which value is not a string?